### ISSUE TITLE
SharedState observe() method and SharedStateCollection for same node

### ISSUE THREAD

**Issue by jipodine** [collective-soundworks/soundworks#69]
The documentation of the `observe`method states:

> The states that are created by the same node are not propagated through the observe callback.

While I understand that their is a use for this, it seems a bit counter-intuitive as the default behaviour. In particular, when moving code from a client (controller, pure observer) to the server (where creation of states may happen), the result changes.

The wide implications of this to the `SharedStateCollection` seem even more bizarre to me. It is not documented, but as most of the methods rely on on underlying `observe`, no locally created state ever appear in them: `find`, `forEach`, `onAttach`, `onUpdate`, etc.

I do not know what would be a good solution
-  an `excludeLocal` parameter (for both the `observe` and the `getCollection` methods.
- a separate `observeOthers` method (and `getCollectionOfOthers`?)

**Comment by b-ma** [comment]
Yup right, I would go for the `excludeLocal` parameter solution, i.e. something like:

```js
StateManager::observe(schemaName, callback, { excludeLocal = false } = {});
StateManager::getCollection(schemaName, { excludeLocal = false } = {});
```

**Comment by b-ma** [comment]
Fixed with d0d255e and d0d255e
